src/thor/astar.cc

```diff
@@ -167,7 +167,7 @@ void AStarPathAlgorithm::ExpandForward(GraphReader& graphreader,
     auto p = destinations_.find(edgeid);
     if (p != destinations_.end()) {
       // Subtract partial cost and time
-      newcost -= p->second;
+      newcost *= p->second;
```

Comment:
this doesn't match the comment, presume you still want -=

Reply:
ah yeah i didnt change the comment i changed the code to store the percentage along the edge instead of the precomputed cost